composer.json

```diff
@@ -18,13 +18,14 @@
     "require": {
         "php": ">=5.3.2",
         "swiftmailer/swiftmailer": ">=4.2.0,~5.0",
-        "symfony/swiftmailer-bridge": "~2.1"
+        "symfony/swiftmailer-bridge": "~2.3"
```

Comment:
Actually, we don't need it anymore, as it only contains a deprecated class we don't use anymore